WakeLock/WakeLock_request_method.html

```diff
@@ -0,0 +1,86 @@
+<div>{{draft}}{{securecontext_header}}{{DefaultAPISidebar("Screen Wake Lock API")}}</div>
+
+<p class="summary">The <strong><code>request()</code></strong> method of the {{domxref("WakeLock")}} interface returns a {{jsref("Promise")}} that when resolved returns a {{domxref("WakeLockSentinel")}} object, which in turn allows control over screen dimming and locking.</p>
+
+
+<h2 id="Syntax">Syntax</h2>
+
+<pre class="syntaxbox">var <var>wakeLock</var> = navigator.wakeLock.request('screen');</pre>
+
+<h3 id="Parameters">Parameters</h3>
+
+<dl>
+  <dt><code>type</code></dt>
+  <dd>A valid wake lock type<br />
+    <table class="standard-table">
```

Comment:
Here's the [standard way of formatting this list](https://wiki.developer.mozilla.org/en-US/docs/Web/API/AbsoluteOrientationSensor/AbsoluteOrientationSensor#Parameters). MDN is remarkably consistent about this.